test/gtest/uct/uct_test.cc

```diff
@@ -461,6 +468,10 @@ std::vector<const resource*> uct_test::enum_resources(const std::string& tl_name
         ucs_async_context_destroy(async);
     }
 
+    if (all_resources.empty()) {
+        UCS_TEST_ABORT("Could not add any resource");
```

Comment:
why needed? do we want to force failure when some resources not found?

Reply:
after adding that filtering, I want to make sure we have at least one interface, to in usability as we cannot run test without any resource, and we get crash otherwise.